.travis.yml

```diff
@@ -0,0 +1,72 @@
+language: php
+
+php:
+  - 7.2
```

Comment:
7.2 doesn't seem necessary any longer? it's in security support until 30 Nov 2020